server/locale/DE/Pi30/mutations/pi30q.js

```diff
@@ -0,0 +1,200 @@
+import Logo from '../../../../message/logos';
+import {
+    xSmallFallback,
+    textWrap,
+    messageLogoWidth,
+    altNoWrap,
+    setLogoTop,
+    xSmallNoWrap,
+    addPeriod
+} from '../../../../message/mediaQueries';
+import { flexLogoMutations, textLogoMutations } from '../../../../message/logoMutations';
+
+const headlineBreaks = [
+    {
+        sizes: ['xsmall'],
+        breaks: ['kaufen,']
+    },
+    {
+        sizes: ['medium'],
+        breaks: ['nach 30']
```

Comment:
```suggestion
        breaks: ['Einkäufen']
```
'Einkäufen' might be a more natural breakpoint to make the message look a bit more even where it breaks.